### PR TITLE
Fix Cannot access built-in declaration 'kotlin.String'. Ensure that you have a dependency on the Kotlin standard library

### PR DESCRIPTION
**Pull request by fzyzcjy**
Closes https://github.com/luanpotter/audioplayers/issues/790

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -47,3 +47,11 @@ allprojects {
         }
     }
 }
+
+dependencies {
+    implementation "androidx.core:core-ktx:+"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+}
+repositories {
+    mavenCentral()
```
Comment (erickzanardo) [inline android/build.gradle:56]: Is this necessary? Maven central is already a default repository for cradle/maven if I remember correctly

**Review by erickzanardo — APPROVED** [review]
One question, but LGTM! Thanks for this PR

**Comment by koji-1009** [comment]
ref #723 #771

This PR addresses issues prior to Kotlin 1.3.
As I wrote in my previous PR, this response shouldn't be necessary...

> - Remove stdlib library from dependency block
>    - ref https://kotlinlang.org/docs/reference/whatsnew14.html#dependency-on-the-standard-library-added-by-default

File: android/build.gradle [hunk 2]
```diff
@@ -47,3 +47,11 @@ allprojects {
         }
     }
 }
+
+dependencies {
+    implementation "androidx.core:core-ktx:+"
```
Comment (koji-1009) [inline android/build.gradle:52]: I think the build will pass without it, because there is no place to use `ktx`.

**Comment by fdietze** [comment]
This fixes #810 for me!

**Review by luanpotter — APPROVED** [review]
LGTM

**Pull request merged**